src/components/Home/UpgradeNotice.tsx

```diff
@@ -38,11 +38,13 @@ function UpgradeNotice({
   canClusterUpgrade,
   onClick,
   className,
+  clusterId,
 }: IUpgradeNoticeProps) {
   if (!canClusterUpgrade) return null;
 
   return (
     <UpgradeWrapperDiv
+      id={`upgrade-notice-${clusterId}`}
```

Comment:
it warned about clusterId not being used, but it most certainly is being used in `mapStateToProps`

So I added it here as an id to kind of trick the linter...  not too happy about that. Any thoughts?

Reply:
Thats a perfectly valid fix :+1: